.github/workflows/gate.yaml

```
@@ -22,7 +22,7 @@ jobs:
         uses: actions/checkout@v3
       - name: Build
         run: |-
-          ./build_product rhel7 rhel8 rhel9 rhel10 --derivatives
+          ./build_product rhel7 rhel8 rhel9 rhel10 al2023 --derivatives
```

```suggestion
          ./build_product rhel7 rhel8 rhel9 rhel10 --derivatives
```
This can be removed unless AL2023 has derivatives like RHEL does with CentOS Stream.